internal/pkg/hub/manifest.go

```diff
@@ -0,0 +1,95 @@
+package hub
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httputil"
+
+	"github.com/confluentinc/cli/internal/pkg/ccstructs"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/log"
+)
+
+func (c *Client) GetRemoteManifest(owner, name, version string) (*ccstructs.Manifest, error) {
+	manifestUrl := fmt.Sprintf("%s/api/plugins/%s/%s", c.URL, owner, name)
```

Comment:
Have you talked to the Connect team about making an API spec / generated SDK for these endpoints?

Reply:
Nope.